src/views/Taiga.vue

```diff
@@ -1,10 +1,55 @@
 <template>
+    <div>
+        <v-layout row>
+            <v-flex xs2>
+                <v-navigation-drawer permanent absolute>
```

Comment:
@HongfeiJu There is a component already made for the sidebar - `NavDrawerTemporary.vue`. Please try using that, instead of creating a new nav bar.

Reply:
I have pulled the last update. For the navigation bar, I just want to follow a similar layout and style.  However, the contents are different. In the taiga view, I added some buttons to control the presence of data, not to execute the route. It doesn't perfectly match with the existing component. So, would it be fine if I just use the current version?

Reply:
@HongfeiJu The reason behind using a *temporary* nav bar instead of a *permanent* one on GitHub and Taiga pages is that we'll need that space to display lots of data. On the dashboard, we don't have that requirement.

Reply:
@amankaushik Added. Please check it. 
![image](https://user-images.githubusercontent.com/35699883/52905946-79b77b00-31ff-11e9-8455-6ca755a78f51.png)